ruby/deepgram-audio-summary/index.rb

```diff
@@ -0,0 +1,42 @@
+require "uri"
+require "net/http"
+require "openssl"
+require "json"
+
+
+def main(req, res)
+  fileUrl = nil
+  begin
+    payload = JSON.parse(req.payload)
+    fileUrl = payload["fileUrl"]
+  rescue Exception => err
+      puts err
+      raise "Payload is invalid."
```

Comment:
let's return the error using `res.json({success: false, message: ...})` instead.